docs/zk2kraft.html

```diff
@@ -15,14 +15,17 @@
  limitations under the License.
 -->
 
-<script><!--#include virtual="js/templateData.js" --></script>
-
-<script id="zk2kraft-template" type="text/x-handlebars-template"></script>
-
+<!--#include virtual="../includes/_header.htm" -->
+<!--#include virtual="../includes/_top.htm" -->
+<div class="content">
 <div class="p-zk2kraft">
-    <h4 class="anchor-heading">Significant Changes in Kafka 4.0 Release</h4>
-    <p>The following are some of the updates in Kafka 4.0 release:</p>
-    <h5 class="anchor-heading">Removal Zookeeper configs</h5>
+    <h1>Differences Between KRaft mode and Zookeeper mode</h1>
```

Comment:
Shall we link to the migration page from this page as well?

Reply:
Nice idea.
I'm thinking about where we should place the link.
Maybe we could rephrase the `Removed Zookeeper Features`:
> This section documents the removed ZooKeeper features and identifies which KRaft features can replace them.
Note that some ZooKeeper features don't need replacements as they were only needed in ZooKeeper.
If you need detailed migration steps, please refer to the **ZooKeeper to KRaft Migration section**.

Reply:
Perhaps (feel free to edit):

```
    <p>
        This section documents differences in behavior between KRaft mode and ZooKeeper mode. More specifically, there are a number of configurations, metrics and features that have been changed or are no longer required in KRaft mode.
<p>
        To migrate an existing cluster from ZooKeeper mode to KRaft mode, please refer to the <a href="/39/documentation.html#kraft_zk_migration">ZooKeeper to KRaft Migration</a> section.
    </p>
```